### DISCUSSION TITLE
How to enable sequence-parallel?

### DISCUSSION THREAD

**Discussion by kback1** [NVIDIA-NeMo/NeMo#6880] [Q&A] [answered]
I tried modifying source to pass argument `sequence-parallel=True` in Transformer Engine [here](https://github.com/kback1/NeMo/blob/main/nemo/collections/nlp/modules/common/megatron/transformer.py/#L848) but that resulted in some issues ([see thread](https://github.com/NVIDIA/TransformerEngine/discussions/283)). How can I set the sequence-parallel argument through NeMo? I am running with:
```
    model.tensor_model_parallel_size=4 \
    model.pipeline_model_parallel_size=2 \
```

**Comment by ksivaman** [comment, marked as answer]
Adding `model.sequence_parallel=True` to the arg list should do the trick.
